ansible/playbook.yml

```diff
@@ -0,0 +1,52 @@
+- name: todoapp
+  hosts: franky-host
+  remote_user: ec2-user
+  vars_prompt:
+    - name: secret_token
+      private: no
+    - name: trello_token
+      private: no
+    - name: token_token
+      private: no
+
+  tasks:
+  - name: ensure git is at the latest version
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: yes # become root user, elevating your priviledge to sudo
+  - name: install python
+    ansible.builtin.yum:
+      name: python3
+      state: latest
```

Comment:
This task needs `become: yes` like the previous yum task. It succeeds if all it does is verify python3 is already installed, but it isn't capable of actually installing it.